cmd/kcn/main.go

```diff
@@ -66,6 +66,9 @@ func init() {
 
 		// See utils/nodecmd/db_migration.go:
 		nodecmd.MigrationCommand,
+
+		// See utils/nodecmd/snapshot.go:
+		nodecmd.SnapshotCommand,
```

Comment:
SnapshotCommand are net required in kscn, kspn and ksen nodes, would it?

Reply:
Snapshot-related functions are not yet considered for the service chain.